Tasks/MSBuild/task.json

```diff
@@ -171,6 +171,11 @@
         "UnableToFindMSBuildVersion0Architecture1LookingForLatestVersion": "Unable to find MSBuild: Version = {0}, Architecture = {1}. Looking for the latest version.",
         "MSBuildNotFoundVersion0Architecture1TryDifferent": "MSBuild not found: Version = {0}, Architecture = {1}. Try a different version/architecture combination, specify a location, or install the appropriate MSBuild version/architecture.",
         "RecordProjectDetailsOnlySupportedOnWindows": "`Record Project Details` is only supported on Windows.",
-        "CreateLogFileOnlySupportedOnWindows": "`Create Log File` is only supported on Windows."
+        "CreateLogFileOnlySupportedOnWindows": "`Create Log File` is only supported on Windows.",
+        "MSB_Build0" : "Build {0}",
+        "MSB_DetectedNuGetExtensionsLoaderPath0": "Detected NuGet extensions loader path. Environment variable NUGET_EXTENSIONS_PATH is set to '{0}'.",
+        "MSB_SolutionNotFoundUsingSearchPattern0": "Solution not found using search pattern '{0}'.",
+        "MSB_RestoreNuGetPackagesDeprecated": "The 'Restore NuGet Packages' option is deprecated. To restore NuGet packages in your build, add a NuGet Installer task to your build definition.",
+        "MSB_BuildToolNotFound": "MSBuild or xbuild(Mono) were not found on the Mac/Linux agent"
```

Comment:
Suggestions:
1. Add a space between "xbuild" and "(Mono)"
2. Change "Mac" to "macOS" with a lowercase "m" for the name of the OS